packages/@react-stately/dnd/src/useDraggableCollectionState.ts

```diff
@@ -62,6 +65,9 @@ export function useDraggableCollectionState(props: DraggableCollectionStateOptio
     preview,
     getAllowedDropOperations
   } = props;
+  if (isDisabled) {
+    selectionManager.isDisabled = () => true;
+  }
```

Comment:
Noticed that doing this makes it so clicking on the Table row itself doesn't cause the row to be selected if drag and drop is disabled. Perhaps useDraggableState should just return `isDisabled` and useDraggableItem should use `state.isDisabled ||  state.selectionManager.isDisabled(props.key)` as its `isDisabled` check instead? Will think about this some more and ask the team for opinions.

Reply:
I'm sorry for the late response. 
I've made some adjustments to the functionality and added new tests!

- [modified the condition of disabled flag](https://github.com/adobe/react-spectrum/pull/5641/commits/123aaeb048694d93ab8da660d541e04d2803999b)
- [Add test for DraggableTable with selection](https://github.com/adobe/react-spectrum/pull/5641/commits/08bcdacb63d7c6a04c62a5f2655198811609b062)